docs/api/authentication.md

```diff
@@ -0,0 +1,47 @@
+<!--[meta]
+section: api
+title: Authentication
+order: 4
+[meta]-->
+
+# Authentication
+
+Authentication strategies allow users to identify themselves to Keystone. This can be used to restrict access to the AdminUI, and to configure [access controls](/guides/access-control/).
+
+- For password logins see: [`auth-password`](/keystone-alpha/auth-password/)
+- For social logins using [Passport.js](http://www.passportjs.org/) see: [`auth-passport`](/keystone-alpha/auth-passport/)
+
+## Usage
+
+```javascript
+const authStrategy = keystone.createAuthStrategy({
```

Comment:
I like to include the imports of things that are the focus of code snippets to ensure no one gets confused when copy+pasta-ing.

```suggestion
const { PasswordAuthStrategy } = require('@keystone-alpha/auth-password');

const authStrategy = keystone.createAuthStrategy({
```